app/views/chargeback/_cb_rate_show.html.haml

```diff
@@ -18,23 +18,20 @@
         %th= _('Group')
         %th= _('Description')
         %th= _('Rate')
-        %th= _('Per Time')
-        %th= _('Per Unit')
+        %th= _('Units')
     %tbody
       - @sb[:selected_rate_details].each do |r|
```

Comment:
We're calling `show_rates` in a loop over each of the details which calls `detail_currency`.  I believe this will be an N+1 query.  Can you confirm this and if so preload the `detail_currency` here before entering the loop?  See [here](http://www.sitepoint.com/silver-bullet-n1-problem/) for more about N+1.